src/coverlet.collector/DataCollection/CoverletSettingsParser.cs

```diff
@@ -131,6 +131,13 @@ private string[] ParseExcludeFilters(XmlElement configurationElement)
                 }
             }
 
+            // if we've only one element mean that we only added CoverletConstants.DefaultExcludeFilter
```

Comment:
wouldn't make that assumption, but more like `excludeFilters.Contains(CoverletConstants.DefaultExcludeFilter)`
is way safer.

Reply:
`CoverletConstants.DefaultExcludeFilter` will be always present